dataset/dataset-import.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+
+set -e
+
+### DATASET PROVIDER - KEYCLOAK REST API SERVICE ###
+
+create_realms () {
+  echo "Creating $1 realm/s "
+  execute_command "create-realms?count=$1"
+}
+
+create_custom_realms () {
+  echo "Creating $1 realm/s with $2 client/s and $3 user/s"
+  execute_command "create-realms?count=$1&clients-per-realm=$2&users-per-realm=$3"
+}
+
+create_clients () {
+  echo "Creating $1 client/s in realm $2"
+  execute_command "create-clients?count=$1&realm-name=$2"
+}
+
+create_users () {
+  echo "Creating $1 user/s in realm $2"
+  execute_command "create-users?count=$1&realm-name=$2"
+}
+
+create_events () {
+  echo "Creating $1 event/s in realm $2"
+  execute_command "create-events?count=$1&realm-name=$2"
+}
+
+create_offline_sessions () {
+  echo "Creating $1 offline sessions in realm $2"
+  execute_command "create-offline-sessions?count=$1&realm-name=$2"
+}
+
+delete_realms () {
+  echo "Deleting realm/s with prefix $1"
+  execute_command "remove-realms?remove-all=true&realm-prefix=$1"
+}
+
+import_status () {
```

Comment:
Can we not use this status command to actually poll for the execution of the command for a default no of tries for a defined set of interval ?

Reply:
As right now these commands, which are wrappers around the data set module, would fail to start if there is an existing command thats in action. It would be nice for this to be self-sufficient to handle that area of the logic

Reply:
Alternatively, we should know what exit code would the functions return, if they are unable to start a task using the dataset module.

Reply:
Example: 
```
$ ./dataset-import.sh -r 2
Creating 2 realm/s 
{"status":"Task started successfully","task-id":"Creation of 2 realms from realm-1 to realm-2, started: Mon Aug 29 19:37:34 UTC 2022","task-status-url":"https://keycloak.192.168.39.41.nip.io/realms/master/dataset/status"}

$ ./dataset-import.sh -r 2
Creating 2 realm/s 
{"error":"Task not triggered. There is already existing task in progress. See task-id for details about existing task","task-id":"Creation of 2 realms from realm-1 to realm-2, started: Mon Aug 29 19:37:34 UTC 2022","task-status-url":"https://keycloak.192.168.39.41.nip.io/realms/master/dataset/status"}

```